processors/pom.xml

```diff
@@ -40,6 +40,12 @@
 			<groupId>com.github.javaparser</groupId>
 			<artifactId>javaparser-core</artifactId>
 		</dependency>
-
+    
+    <dependency>
```

Comment:
Would be nice to add the dependency to the pluginManagement block in the parent pom, using a property for the version (like other dependencies in the project). Also to fix indentation ;)

Reply:
@adrienbaron Strange, but I see indentation as normal, looks like Tab used in pom.xml :-(